PubNub/Data/Managers/PNSubscriber.m

```diff
@@ -1225,7 +1226,11 @@ - (void)appendSubscriberInformation:(PNStatus *)status {
     status.subscribedChannelGroups = [_channelGroupsSet allObjects];
 }
 
-#pragma mark -
+#pragma mark - Internal
+- (void)didEnterBackground {
+    [self.client.heartbeatManager stopHeartbeatIfPossible];
+    [self stopRetryTimer];
```

Comment:
I am not quite sure about how it works but when I set a breakpoint at startHeartbeat it get called when the game comes back. The stack trace shows this is triggered by some callback. 
QA has verified the functionality works fine. Yeah I don't know if there is a better solution to this problem.